tests/ui/specs/Input/Input-specs.js

```diff
@@ -0,0 +1,645 @@
+		});
+
+		it('should focus Overlay BackButton', () => {
+			Page.spotlightDown();
+			Page.spotlightRight();
+			Page.spotlightRight();
+			Page.spotlightRight();
+			Page.spotlightRight();
+			Page.spotlightSelect();
+			Page.spotlightUp();
+
+			expect(components.input10.backButton.isFocused()).to.be.true();
+		});
+
+		it('should exit Overlay Input using 5-way enter', () => {
+			Page.spotlightDown();
+			Page.spotlightRight();
+			Page.spotlightRight();
+			Page.spotlightRight();
+			Page.spotlightRight();
+			Page.spotlightSelect();
+			Page.spotlightSelect();
+
+			expect(components.input10.self.isFocused()).to.be.true();
+			expect(components.input10.self.getText()).equals('placeholder');
+		});
+
+		it('should exit Overlay Input using BackButton', () => {
+			Page.spotlightDown();
+			Page.spotlightRight();
+			Page.spotlightRight();
+			Page.spotlightRight();
+			Page.spotlightRight();
+			Page.spotlightSelect();
+			Page.spotlightUp();
+			Page.spotlightSelect();
+
+			expect(components.input10.self.isFocused()).to.be.true();
+			expect(components.input10.self.getText()).equals('placeholder');
+		});
+	});
+
+	describe('using pointer', () => {
+		// Text Input
+		it('should open Text Input on click', () => {
+			components.input1.self.click();
+			components.input1.inputField.click();
+
+			expect(components.input1.title.getText()).equals('Text Input');
+			expect(components.input1.inputField.isFocused()).to.be.true();
+		});
+
+		it('should exit Text Input using BackButton', () => {
+			components.input1.self.click();
+			components.input1.background.click();
+			components.input1.backButton.click();
+			Page.spotlightSelect();
+
+			expect(components.input1.self.isFocused()).to.be.true();
+			expect(components.input1.self.getText()).equals('placeholder');
+		});
+
+		// Password Input
+		it('should open Password Input on click', () => {
+			components.input2.self.click();
+			components.input2.inputField.click();
+
+			expect(components.input2.title.getText()).equals('Password Input');
+			expect(components.input2.inputField.isFocused()).to.be.true();
+		});
+
+		it('should exit Password Input using BackButton', () => {
+			components.input2.self.click();
+			components.input2.background.click();
+			components.input2.backButton.click();
+			Page.spotlightSelect();
+
+			expect(components.input1.self.isFocused()).to.be.true();
+			expect(components.input1.self.getText()).equals('placeholder');
+		});
+
+		// Number Input
+		it('should open Number Input on click', () => {
+			components.input3.self.click();
+			components.input3.numberButton.click();
+
+			expect(components.input3.numberCell.getText()).equals('1');
+			expect(components.input3.title.getText()).equals('Number Input');
+		});
+
+		it('should exit Number Input using BackButton', () => {
+			components.input3.self.click();
+			components.input3.background.click();
+			components.input3.backButton.click();
+			Page.spotlightSelect();
```

Comment:
Note for the next reviewer: The Input button is not automatically refocused when the button is closed via backButton click. This might be a bug